client/my-sites/customer-home/cards/education/earn/index.jsx

```diff
@@ -1,24 +1,40 @@
 import { useTranslate } from 'i18n-calypso';
 import { connect } from 'react-redux';
 import earnCardPrompt from 'calypso/assets/images/customer-home/illustration--secondary-earn.svg';
+import VideoModal from 'calypso/components/videos-ui/video-modal';
+import { COURSE_SLUGS } from 'calypso/data/courses';
+import { useRouteModal } from 'calypso/lib/route-modal';
 import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 import EducationalContent from '../educational-content';
-
 export const EDUCATION_EARN = 'home-education-earn';
 
 const EducationEarn = ( { siteSlug } ) => {
 	const translate = useTranslate();
+	const { isModalOpen, openModal, closeModal } = useRouteModal(
+		'coursePaymentsModal',
```

Comment:
Any reason why we can use the `courseSlug` param we use for the other course?

Reply:
Yes, there is a reason. If you have 2 or more modals on the same page those slugs must be different. Otherwise the same last modal will be opened. Slug is query param identifier.

Reply:
I think I didn't explain myself very well on my initial comment, sorry. I'm not asking about the slugs, they definitely should be different. But I don't understand why we're setting a different param name here.

It seems to me that it should be like this for consistency with the other course:
```suggestion
		'courseSlug',
```

Reply:
Please take a look  at [useRouteModal](https://github.com/Automattic/wp-calypso/blob/d244c69d7c30a3b318cadde39a702263c9bb845b/client/lib/route-modal/use-route-modal.tsx#L22).
First param - queryKey is identifier for modal, no matter of value.

Reply:
That's super confusing and sounds like a bug that would be nice to address on a follow up 😄 . Not a blocker for this PR though